AUTHORS.md

```diff
@@ -20,6 +20,35 @@ The rules for this file:
   * Please start a new section for each new year
   * Don't ever delete anything
 -->
+**2016**
+- Matteo Tiberti <matteo.tiberti@gmail.com>
+- Tone Bengtsen <tonebengsten@gmail.com>
+- Wouter Boomsma <wb@bio.ku.dk>
+- Richard Gowers <richardjgowers@gmail.com>
+- Oliver Beckstein <orbeckst@gmail.com>
+- Max Linke <max_linke@gmx.de>
+- Jonathan Barnoud <jonathan@barnoud.net>
+
+**2018**
+- Tyler Reddy <tyler.je.reddy@gmail.com>
+- Zeman <zeman@icp.uni-stuttgard.de>
+
+**2019**
+- Matthijs Tadema <M.J.Tadema@protonmail.com>
+
+**2020**
+- Rocco Meli <r.meli@bluemail.ch>
+- shfrz <@shfrz>
+- Irfan Alibay <@IAlibay>
+
+**2021**
+- Philip Loche <ploche@physik.fu-berlin.de>
+- Paarth Thadani <paarth.thadani15@gmail.com>
```

Comment:
Just having a quick look through there's a co-authored commit by @lilyminium on the related PR: https://github.com/MDAnalysis/mdanalysis/pull/3178/commits/bf0310e40edbcd8d40cbef482503a5a5a0f16052

Should she be added too?

Reply:
oof, yes. I was going by the commit authors, let me make sure to add all co-authors

Reply:
Looks like she was the only missing author, thanks for pointing that out!